man/ukify.xml

```diff
@@ -671,8 +671,8 @@
       --phases='enter-initrd:leave-initrd enter-initrd:leave-initrd:sysinit \
                 enter-initrd:leave-initrd:sysinit:ready' \
       --pcr-banks=sha384,sha512 \
-      --secureboot-private-key=sb.key \
-      --secureboot-certificate=sb.cert \
+      --secureboot-private-key=secure-boot-key.pem \
+      --secureboot-certificate=secure-boot-certificate.pem \
```

Comment:
If the file names are changed, then I'd like to make them consistent with the option name, i.e. secureboot-private-key.pem and secureboot-certificate.pem.